### DISCUSSION TITLE
Add dummy layers at end of print?

### DISCUSSION THREAD

**Discussion by GWR57xx** [sn4k3/UVtools#650] [Q&A] [answered]
Just like the initial dummy first layer at the start of a print job, is it possible to add dummy last layers at the end of a print job?
Reason for asking is I have a Mars 3 which always runs to its highest Z position at the end of a run, resulting in splatter all over the machine from resin dripping back into the vat off the build plate.
I would like to add a last layer that raises the build plate only as high as it needs to go to raise the printed item out of the resin in the vat, and hold this position for (say) 20 minutes.
Thanks for reading this far.

**Comment by sn4k3** [comment, marked as answer]
Check this tool:
![image](https://user-images.githubusercontent.com/113281/214620785-7034ad4a-8f0c-45b2-966b-66d02e1a126e.png)

**Reply by GWR57xx** [reply]
That's perfect, thank you! Does exactly what I want.
I feel such a fool for not finding that option before.
I think I was looking for something more complicated...
